development/kops/run_sonobuoy.sh

```diff
@@ -43,6 +43,8 @@ do
   echo 'Waiting for the cluster to be ready...'
 done
 
+results=$(./sonobuoy --context ${KOPS_CLUSTER_NAME} retrieve)
+./sonobuoy e2e --wait $results --rerun-failed --kube-conformance-image k8s.gcr.io/conformance:v1.19.6
```

Comment:
Are we going to need to change this for 1.19.8?

Reply:
We should check the result and loop similar to what we do above up to like 3 times retrying. And yeah remove this hardcoded image, use the var from above.